### ISSUE TITLE
Add option to disable either simplified or traditional characters in popup

### ISSUE THREAD

**Issue by cwfoo** [leezu/zhongwen#22]
Consider adding an option for disabling the display of either simplified or traditional characters in the popup. 

Suppose I wish to learn simplified characters (possibly after learning traditional characters), it would be great if I can increase focus by avoiding reminders of the traditional characters.  This would help in maintaining a strict mental separation between traditional and simplified characters.

I know that the indiscriminate mixing of characters is not a big problem in greater China, but in the place where I live, it is common for people mix characters indiscriminately (within the same sentence!). I consider this bad style. An option to disable simplified or traditional characters in the popup would help immensely in learning how to write properly.

**Comment by leezu** [comment]
Fixed by https://github.com/leezu/zhongwen/commit/eb2403fbf89984d42adb121e78a46c81215928f7
Thanks @CNurt for the fix and @cwfoo for reporting!
I just made a new release containing the fixes.

**Comment by leezu** [comment]
Hi @cwfoo , could you share some links to websites that you quoted

> where I live, it is common for people mix characters indiscriminately (within the same sentence!). I consider this bad style. An option to disable simplified or traditional characters in the popup would help immensely in learning how to write properly.

**Comment by cwfoo** [comment]
@leezu  It is surprisingly difficult to find citable sources for that phenomenon, so you will have to take my word for it.

Observations on the use of Chinese characters in Malaysia and Brunei (and probably Singapore):

- When people write notes in Chinese, they write in a mixture of simplified and traditional characters out of convenience. The simplified form is easier to write, but there are a substantial number of people who can't remember the simplified characters, so they fall back on traditional characters.

- Random mixture is surprisingly common in restaurant menus. I've also seen this in high end restaurants. The mixture usually occurs between entries, but there are times when the mixture is within the same entry (for example, "雞饭").

Something else you might find interesting:

On Borneo, the most popular Chinese newspapers are Sin Chew Daily (星洲日報), See Hua Daily News (詩華日報), and United Daily News (聨合日報). In their printed editions, they use traditional characters for all headings, and simplified characters for the body. See this:

![An article from Sin Chew Daily](https://user-images.githubusercontent.com/11555232/52169603-79869c80-2776-11e9-9caf-0540bfb9e167.jpg)
([Source](http://www.rmp.gov.my/arkib-berita/keratan-akhbar/2016/06/08/polis-tumpaskan-geng-samun-stesen-minyak). Fair use: to illustrate the point in question).

In summary, I cannot learn to write "properly" by relying on the environment for reminders and validation. That's why the feature implemented in #23 is personally important.
